embeddings/config/flair_config.py

```diff
@@ -0,0 +1,175 @@
+from abc import ABC
+from dataclasses import dataclass, field
+from itertools import chain
+from types import MappingProxyType
+from typing import Any, ClassVar, Dict, Mapping, Set, Tuple, Union
+
+from embeddings.config.base_config import AdvancedConfig, BasicConfig
+from embeddings.data.io import T_path
+from embeddings.utils.utils import read_yaml
+
+
+@dataclass
+class FlairTextClassificationConfigMapping:
+    LOAD_MODEL_KEYS_MAPPING: ClassVar[Mapping[str, Set[str]]] = MappingProxyType(
+        {
+            "FlairDocumentCNNEmbeddings": {
+                "hidden_size",
+                "rnn_type",
+                "rnn_layers",
+                "bidirectional",
+                "dropout",
+                "word_dropout",
+                "reproject_words",
+            },
+            "FlairDocumentRNNEmbeddings": {
+                "cnn_pool_kernels",
+                "dropout",
+                "word_dropout",
+                "reproject_words",
+            },
+            "FlairTransformerDocumentEmbedding": {"pooling", "fine_tune"},
+            "FlairDocumentPoolEmbedding": {"pooling", "fine_tune_mode"},
+        }
+    )
+
+    @classmethod
+    def map_load_model_keys(cls, document_embedding_cls: str) -> Set[str]:
+        return cls.LOAD_MODEL_KEYS_MAPPING[document_embedding_cls]
+
+    @classmethod
+    def get_load_model_keys(cls) -> Set[str]:
+        return set(chain(*cls.LOAD_MODEL_KEYS_MAPPING.values()))
+
+
+@dataclass
+class FlairBasicConfig(BasicConfig, ABC):
```

Comment:
what do you mean? these are shared between these two configs, similarly to `FlairModelTrainerConfigSpace`

Reply:
Ok, these are similar but not identical cases. 

In case of Configs these parameters values could be different as we will set it up after HPS and we do not make any assumption that they are identical. Therefore e.g. mini_batch_size could be set differently for different tasks. We could stay at this version, and we can adapt it later.